tests/performance/README.md

```diff
@@ -15,8 +15,9 @@ $ pip3 install -r ../requirements.txt
 ```
 
 ```bash
-$ docker run -p 9178:9178 openvino/model_server:latest --model_name resnet \
---model_path gs://ovms-public-eu/resnet50 --port 9178 --batch_size 2
+$ wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.{xml,bin} -P models/resnet50/1
+$ docker run -u $(id -u) -v $(pwd)/models:/models -p 9178:9178 openvino/model_server:latest --model_name resnet \
+--model_path /models/resnet50 --port 9178 --batch_size 2
```

Comment:
after update test fails with:
  File "grpc_latency.py", line 148, in <module>
    result = stub.Predict(request, 10.0)
  File "/tmp/pytest-of-k8sworker/pytest-571/test_commands_ovms_tests_perfo0/model_server/tests/performance/.venv/lib/python3.8/site-packages/grpc/_channel.py", line 923, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/tmp/pytest-of-k8sworker/pytest-571/test_commands_ovms_tests_perfo0/model_server/tests/performance/.venv/lib/python3.8/site-packages/grpc/_channel.py", line 826, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.INVALID_ARGUMENT
	details = "Missing input with specific name - Required input: 0"
t: 	debug_error_string = "{"created":"@1673614541.069949285","description":"Error received from peer ipv6:[::1]:31666","file":"src/[...]re/lib/surface/call.cc","file_line":1062,"grpc_message":"Missing input with specific name - Required input: 0","grpc_status":3}"
>